specification/saas/resource-manager/Microsoft.SaaS/preview/2018-03-01-beta/saas.json

```diff
@@ -0,0 +1,866 @@
+            "schema": {
+              "$ref": "#/definitions/AccessTokenResult"
+            }
+          },
+          "default": {
+            "description": "A response indicating an error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "generated SaaS resource token": {
+            "$ref": "./examples/saasV2/ListAccessTokenPost.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Result": {
+      "description": "Sample result definition",
+      "properties": {
+        "sampleProperty": {
+          "type": "string",
+          "description": "Sample property of type string"
+        }
+      }
+    },
+    "OperationListResult": {
+      "description": "Operation List Result Response",
+      "properties": {
+        "value": {
+          "description": "List of SaaS operations supported by the SaaS resource provider.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Operation"
+          }
+        }
+      }
+    },
+    "Operation": {
+      "description": "SaaS REST API operation definition.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "Operation name",
+          "type": "string"
+        },
+        "display": {
+          "description": "Display metadata associated with the operation.",
+          "properties": {
+            "provider": {
+              "description": "Service provider: Microsoft SaaS.",
+              "type": "string"
+            },
+            "resource": {
+              "description": "Resource on which the operation is performed.",
+              "type": "string"
+            },
+            "operation": {
+              "description": "Type of the operation: get, read, delete, etc.",
+              "type": "string"
+            },
+            "description": {
+              "description": "Description of the operation.",
+              "type": "string"
+            }
+          }
+        }
+      }
+    },
+    "SaasResource": {
+      "description": "SaaS REST API resource definition.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "description": "saas properties",
+          "type": "object",
+          "allOf": [
+            {
+              "$ref": "#/definitions/SaasProperties"
+            },
+            {
+              "$ref": "#/definitions/SaasCreationProperties"
+            }
+          ]
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the resource"
+        },
+        "id": {
+          "description": "The resource uri",
+          "type": "string"
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type."
+        },
+        "tags": {
```

Comment:
>tags [](start = 9, length = 4)

You have the `Microsoft.SaaS/saasresources` resource type modeled as a tenant resource, which is a proxy a proxy resource, which doesn't support tags the same way tracked resources do.

Reply:
We use this property for Subscription level resources,
for tenant level resources its always null.